src/components/NavBar/index.tsx

```diff
@@ -47,11 +47,19 @@ const MenuItem = ({ href, dataTestId, id, isActive, children }: MenuItemProps) =
       style={{ textDecoration: 'none' }}
       data-testid={dataTestId}
     >
-      {children}
+      <MenuItemText>{children}</MenuItemText>
     </NavLink>
   )
 }
 
+const MenuItemText = styled.span`
```

Comment:
This would also be a good time refactor the `NavLink` to not use vanilla extract and to handle text wrapping